spec/player_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rspec'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+
+RSpec.describe Player do
+  before(:each) do
+    card1 = Card.new(:spade, 'Ace', 14)
+    card2 = Card.new(:heart, '3', 3)
+    card3 = Card.new(:diamond, '8', 8)
+    cards = [card1, card2, card3]
+    @deck = Deck.new(cards)
+    @player = Player.new('John', @deck)
+  end
```

Comment:
Great job including a `before(:each)` block here to clean up your code and make it more DRY.